application/apps/protocol/binding/proto/src/lib.rs

```diff
@@ -0,0 +1,113 @@
+///
+/// # Entities:
+///
+/// * `Range`
+/// * `RangeInclusive`
+///
+pub mod common {
+    include!(concat!(env!("OUT_DIR"), "/common.rs"));
```

Comment:
I don't know if the backslash will be working on Windows, I think we can test that on Windows first before adjusting it manually. Hopefully the rust compiler can resolve automatically.